CHANGES.md

```diff
@@ -1,3 +1,8 @@
+Airbase 127
```

Comment:
126 hasn't been released yet, so I need to replace the `Plugin downgrades` section in Airbase 126.

Reply:
Maybe we should release 126 first, so that we have the other changes delivered to airbase users?

we should test the surefire upgrade on the trino, ideally before merging this pr

Reply:
So I ran tests on trino-main more than 10 times today (it takes about 15 minutes) and everything was fine. I did manage to reproduce the issue just once, after running it about 6 times, with airbase 125.

Reply:
I published a SNAPSHOT release with this PR as https://oss.sonatype.org/content/repositories/snapshots/io/airlift/airbase/126-SNAPSHOT/airbase-126-20220608.202754-1.pom

you can run full Trino CI with this.

Reply:
Great, thanks! https://github.com/trinodb/trino/pull/12758

Reply:
I rebased and updated this file.